src/lune/builtins/luau.rs

```diff
@@ -0,0 +1,73 @@
+use mlua::prelude::*;
+use mlua::Compiler as LuaCompiler;
+
+use crate::lune::lua::table::TableBuilder;
+// use  as LuaNumber;
+
+pub fn create(lua: &'static Lua) -> LuaResult<LuaTable> {
+    TableBuilder::new(lua)?
+        .with_function("compile", compile_source)?
+        .with_function("load", load_source)?
+        .build_readonly()
+}
+
+fn compile_source<'a>(
+    lua: &'static Lua,
+    (source, options): (LuaString<'a>, Option<LuaTable<'a>>),
+) -> LuaResult<LuaString<'a>> {
+    let mut optimization_level = 1;
+    let mut coverage_level = 0;
+    let mut debug_level = 1;
+
+    if let Some(options) = options {
```

Comment:
Can we split these options out into a full options struct? I think the most straight-forward example we have of that right now is https://github.com/filiptibell/lune/blob/main/src/lune/lua/fs/options.rs , we could do something very similar here like `LuauCompileOptions`, where `nil` becomes a default struct

Reply:
Addressed in; https://github.com/filiptibell/lune/pull/82/commits/b4ff4a5b06076b9e552f2c36c7c5710c14e7489c